### PR TITLE
Add SixPMTest with implementations of WebDriverWait

### PR DESCRIPTION
**Pull request by Shake-Ghairbyan**
Add `Utils` class to keep `getRandomNumber(int max)` and `randomItem(List<WebElement> list)` methods there.
Add `testSuiteSixPMTest.xml` test suite.
Add `WebDriverWait` to `test6PMMainFunctionalities()` in `SixPMTest`.

### CODE REVIEW

File: src/test/java/SixPMTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        Actions actions = new Actions(driver);
+
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+
+        By accessoriesButtonLoc = By.xpath("//*[text()=\"Accessories\"]");
+        WebElement accessoriesButton = driver.findElement(accessoriesButtonLoc);
+        wait.until(ExpectedConditions.elementToBeClickable(accessoriesButtonLoc));
+        actions.moveToElement(accessoriesButton).perform();
+
+        By aviatorLoc = By.xpath("//*[text()=\"Aviators\"]");
```
Comment (David91) [inline src/test/java/SixPMTest.java]: just write //*[text()='Aviators'] instead of this \"

File: src/test/java/SixPMTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertEquals(actualName, expectedName, "Name of the item doesn't match the expected value.");
+        softAssert.assertEquals(actualPrice, expectedPrice, "Price of the item doesn't match the expected value.");
+        softAssert.assertAll();
+
+        WebElement quantityButton = driver.findElement(By.name("quantity"));
+        actions.moveToElement(quantityButton).click().build().perform();
+
+        WebElement removeOption = driver.findElement(By.xpath("//*[text()='Remove']"));
+        removeOption.click();
```
Comment (David91) [inline src/test/java/SixPMTest.java]: This is post test action. Move it in @After... method

File: src/test/java/SixPMTest.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+        String textCheck = "Nothing to see here yet! Sign in to see items that you've previously placed in your Bag or check out all the awesome things you can buy on 6pm!";
+        wait.until(ExpectedConditions.textToBe(By.xpath("//h1/following-sibling::p"), textCheck));
```
Comment (David91) [inline src/test/java/SixPMTest.java]: No need to have this text, it's error prone.
Just check that the product gets invisible in shopping bag after removal.

**Review by David91 — CHANGES_REQUESTED** [review]

File: src/test/java/SixPMTest.java (hunk 3 above)
Comment (Shake-Ghairbyan) [reply src/test/java/SixPMTest.java]: Just to ensure that there is no such item after its removal?

File: src/test/java/SixPMTest.java (hunk 1 above)
Comment (Shake-Ghairbyan) [reply src/test/java/SixPMTest.java]: Addressed in 4a76f79.

File: src/test/java/SixPMTest.java (hunk 2 above)
Comment (Shake-Ghairbyan) [reply src/test/java/SixPMTest.java]: May I have it in `@AfterTest` or it should be in `@AfterMethod`? 
Can you elaborate on the differences between those two?

Comment (David91) [reply src/test/java/SixPMTest.java]: I left it on your convenience.
And the differences between these two you can read in TestNG documentation. It is clearly described there.

File: src/test/java/SixPMTest.java (hunk 3 above)
Comment (David91) [reply src/test/java/SixPMTest.java]: Of course.

**Pull request merged**